tools/hive-bigquery/utilities.py

```diff
@@ -0,0 +1,81 @@
+"""Provides utility functions to other modules"""
+
+import logging
+import random
+import string
+import subprocess
+
+import tableprint as tp
+
+logger = logging.getLogger('Hive2BigQuery')
+
+
+def calculate_time(start, end):
+    """Pretty prints the time taken for an operation
+
+    Args:
+        start (float): Start time of an operation
+        end (float): End time of an operation
+
+    Returns:
+        str: Pretty format the time taken for an operation
+    """
+
+    time_taken = int(round((end - start), 0))
+    day = time_taken // 86400
+    hour = (time_taken - (day * 86400)) // 3600
+    minutes = (time_taken - ((day * 86400) + (hour * 3600))) // 60
+    seconds = time_taken - ((day * 86400) + (hour * 3600) + (minutes * 60))
+    if day != 0:
```

Comment:
Please avoid multiple returns. Set a variable to a value conditionally and do one return at the end